contrib/kube-prometheus/jsonnet/kube-prometheus/alerts/general.libsonnet

```diff
@@ -0,0 +1,34 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'general.rules',
+        rules: [
+          {
+            alert: 'TargetDown',
+            annotations: {
+              description: '{{ $value }}% of {{ $labels.job }} targets are down.',
+              summary: 'Targets are down',
+            },
+            expr: '100 * (count(up == 0) BY (job) / count(up) BY (job)) > 10',
+            'for': '10m',
+            labels: {
+              severity: 'warning',
+            },
+          },
+          {
+            alert: 'DeadMansSwitch',
```

Comment:
We want to definitely have the `TargetDown`, but not necessarily the `DeadMansSwitch`. Can we separate these?

Reply:
Hmm I would prefer to keep it. The kube-prometheus stack is a lot about our opinionated way of doing things due to our experience. And in our experience this is an invaluable tool that I cannot recommend highly enough. Aside from that the kube-prometheus stack by default routes it to /dev/null in the alertmanager, so is this really a problem?

I’m fine with having a way to filter it out though, but would prefer if that’s the explicit action rather than having to explicitly add it.

Reply:
Moving forward with this for now, but more than happy to change things in follow ups.